### ISSUE TITLE
Describe how to set up fineee-grained token for normal access

### ISSUE THREAD

**Issue by quazar-omega** [slapperwan/gh4a#1411]
There are a lot of settings to choose from in the fine-grained token and it's not immediately clear what is needed by Octodroid to function fully, I think it would be great if it had been explained in the app's login pop-up when selecting the token method or in the README, with a link to it in the pop-up.

<details>
<summary>
Currently I tired the following and it fails to load my notifications as well as the news feed:
</summary>

- User permissions:
  - Read access to:
    - gpg keys
    - plan
    - private repository invitations
    - user events
  - Read and Write access to:
    - blocking
    - email addresses
    - followers
    - gists
    - interaction limits
    - profile
    - starring
    - watching
- Repository permissions
  - Read access to:
    - Dependabot alerts
    - commit statuses
    - metadata
  - Read and Write access to:
    - administration
    - code
    - deployments
    - discussions
    - environments
    - issues
    - merge queues
    - pull requests
    - repository custom properties

</details>

**Comment by APT37** [comment]
Indeed, please describe all the permissions needed by the app. Thank you.
